tests/postgres/scan_test.go

```diff
@@ -967,26 +969,25 @@ func TestScanIntoCustomBaseTypes(t *testing.T) {
 		ReplacementCost MyFloat64
 		Rating          *model.MpaaRating
 		LastUpdate      MyTime
-		SpecialFeatures *MyString
+		SpecialFeatures MyStringArray
```

Comment:
If `SpecialFeatures` is pq.StringArray you wouldn't need to exclude SpecialFeatures column bellow.

Reply:
With the type alias like so:
```
type MyStringArray pq.StringArray
```

The MyStringArray type does not implement Scanner. The QRM will then see it as a slice because that's what `pq.StringArray` is and try to apply it's row mapping into the slice. Therefore it will always be null.

Reply:
No, I'm meant without defining a type, either by realiasing or not using new type at all:

```golang
type MyStringArray = pq.StringArray // note '=', MyStringArray now retains Scan method
```
or
```golang
	type film struct {
		FilmID          MyUint16 `sql:"primary_key"`
		Title           MyString
		Description     *MyString
		ReleaseYear     *MyInt16
		LanguageID      MyUint8
		RentalDuration  MyUint8
		RentalRate      MyFloat32
		Length          *MyUint32
		ReplacementCost MyFloat64
		Rating          *model.MpaaRating
		LastUpdate      MyTime
		SpecialFeatures *MyString
		SpecialFeatures pq.StringArray
	}
```